src/stop_mode.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "locomotion_mode/locomotion_mode.hpp"
+
+namespace locomotion_mode {
+
+  class StopMode : public LocomotionMode
+  {
+  public:
+    StopMode(rclcpp::NodeOptions options, std::string node_name);
+
+  private:
+    rover_msgs::msg::JointCommandArray compute_joint_commands(const geometry_msgs::msg::Twist::SharedPtr msg);
+  };
+}
```

Comment:
```suggestion
namespace locomotion_mode {

class StopMode : public LocomotionMode
{
public:
  StopMode(rclcpp::NodeOptions options, std::string node_name);

private:
  rover_msgs::msg::JointCommandArray compute_joint_commands(const geometry_msgs::msg::Twist::SharedPtr msg);
};
}
```

Remove indentation level inside of namespace according to style guide